ci-operator/step-registry/hypershift/azure/run-e2e/hypershift-azure-run-e2e-commands.sh

```diff
@@ -15,13 +15,30 @@ trap cleanup EXIT
 
 export EVENTUALLY_VERBOSE="false"
 
+EXTERNAL_DNS_ARGS=""
+if [[ "${HYPERSHIFT_EXTERNAL_DNS_DOMAIN:-}" != "" ]]; then
+  EXTERNAL_DNS_ARGS="--e2e.external-dns-domain=${HYPERSHIFT_EXTERNAL_DNS_DOMAIN}"
+fi
+
+AKS_ANNOTATIONS=""
+if [[ "${AKS}" == "true" ]]; then
+AKS_ANNOTATIONS="--e2e.annotations hypershift.openshift.io/pod-security-admission-label-override=baseline \
+  --e2e.annotations hypershift.openshift.io/certified-operators-catalog-image=registry.redhat.io/redhat/certified-operator-index@sha256:fc68a3445d274af8d3e7d27667ad3c1e085c228b46b7537beaad3d470257be3e \
```

Comment:
@fxierh - didn't you have a way of extracting these from the release image? We should do that rather than hardcoding them here.

Reply:
yeah there is a command for retrieving these I will update the pr now https://github.com/openshift/release/blob/6241daf293a2d892960aabb6dc928cc4b1a28d1d/ci-operator/step-registry/cucushift/hypershift-extended/k8s-mgmt/get-guest-annotations/cucushift-hypershift-extended-k8s-mgmt-get-guest-annotations-commands.sh

Reply:
Oh, I missed that somehow. Thanks for bringing it up @bryan-cox !

Reply:
There's a catch though: the 4.17 images e.g. registry.redhat.io/redhat/certified-operator-index:v4.17 do not exist yet. We may use the 4.16 images for now and switch to the 4.17 images when they become available.